boards/arm/pinnacle_100_dvk/pinnacle_100_dvk_defconfig

```diff
@@ -0,0 +1,40 @@
+# Copyright (c) 2020 Laird Connectivity
+
+# SPDX-License-Identifier: Apache-2.0
+
+CONFIG_SOC_SERIES_NRF52X=y
+CONFIG_SOC_NRF52840_QIAA=y
+CONFIG_BOARD_PINNACLE_100_DVK=y
+
+# Enable MPU
+CONFIG_ARM_MPU=y
+
+# Enable RTT
+CONFIG_USE_SEGGER_RTT=y
+
+# enable GPIO
+CONFIG_GPIO=y
+
+# enable uart driver
+CONFIG_SERIAL=y
+
+# enable console
+CONFIG_CONSOLE=y
+CONFIG_UART_CONSOLE=y
+
+# additional board options
+CONFIG_GPIO_AS_PINRESET=y
+
+# Modem driver
+CONFIG_MODEM=y
+CONFIG_MODEM_RECEIVER=y
+CONFIG_MODEM_HL7800=y
+
+# Networking
+CONFIG_NETWORKING=y
```

Comment:
Are you sure you want networking enabled by default? You could instead leave it to the user

Reply:
We feel the point of using our modem is to use it for networking, thus enabling it by default to make it easier for the user out of the gate.

Reply:
I will block this PR because of `CONFIG_NETWORKING=y` and `CONFIG_NET_IPV6=n`, which leads to unnecessary changes in tests and samples. This board is based on nRF52840 and therefore also supports Bluetooth LE, 802154, and IPv6. If nRF52840 does not have any RF functionality, then the items "ble, ieee802154" must be removed. 

And in general it is not necessary to enable CONFIG_NETWORKING on board level. Which networking sample does not do it anyway?